src/aiodynamo/client.py

```diff
@@ -669,6 +685,7 @@ async def send_request(
             if failed is not None:
                 raise failed
             raise
+        raise BrokenThrottleConfig()
```

Comment:
Is this to somehow handle empty generator throttle, or to silence mypy?

Reply:
primarily: silence mypy. However, I realized that if a custom throttle config for some reason immediately exits, it would actually also reach this point and indicate broken code, so raising an exception seemed reasonable after all here.